.github/workflows/builder_go_slsa3.yml

```diff
@@ -48,12 +55,17 @@ on:
         type: string
         default: ".slsa-goreleaser.yml"
       evaluated-envs:
-        description: "Evaluated env variables to pass to the builder"
+        description: "Evaluated env variables to pass to the builder."
         required: false
         type: string
+      compile-builder:
+        description: "Build the builder from source. This speeds up build time by ~2mn."
```

Comment:
This increases the build time?

Reply:
no it does not ^^. Thanks for catching this